src/main/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/gencode/GencodeFuncotation.java

```diff
@@ -187,59 +200,68 @@ public String serializeToVcfString() {
 
     @Override
     public void setFieldSerializationOverrideValue( final String fieldName, final String overrideValue ) {
```

Comment:
this method and the next method and the override variables feel like they could maybe be replaced with a map instead of hardcoded cases?  Is there a reason to do it this way?

Reply:
In the initial implementation there was no ability to override fields, so every field had its own variable. 

I agree - now that there are overrides, it makes more sense to replace all the variables with a map so that they can have their fields overridden easily.